Aztec/Classes/TextKit/TextView.swift

```diff
@@ -983,7 +1095,21 @@ open class TextView: UITextView {
     open func toggleBlockquote(range: NSRange) {
         ensureInsertionOfEndOfLineForEmptyParagraphAtEndOfFile(forApplicationRange: range)
 
-        let formatter = BlockquoteFormatter(placeholderAttributes: typingAttributes)
+        //track nested depth
+        let paragraphStyle = typingAttributes.paragraphStyle()
+        let currentDepth = paragraphStyle.blockquoteNestedIndent.depth
+        
+        //get original color or previous nested color as appropriate when toggling back and forth
+        var originalQuoteAttributes = typingAttributes
+        originalQuoteAttributes.removeValue(forKey: .foregroundColor)
+        
+        let textColor = getTextColorForQuoteDepth(currentDepth, toggle: true)
+        originalQuoteAttributes[.foregroundColor] = textColor.prev
+        
+        let formatter = BlockquoteFormatter(placeholderAttributes: originalQuoteAttributes)
+        //use the first color when toggling on
+        formatter.nextTextColor = textColor.next
```

Comment:
At the moment I don't think we should change the color of the text in the blockquote. I understand that you have a need for this for your work, but can we handle this in a separate follow up PR?

The color changes need to be done inside the formatter and we will need to send the value of color to apply/revert to the formatter, but again I think that change should be done on a separate PR and we should focus on the border color on this one.

Reply:
ok, I will separate that out into another PR